dcos/dcos-ui
# DCOS-15073: add external volume system test

This PR adds a new system test that creates a single container service with external volume and tests if the volume is persistent after the service has been suspended and resumed. A separate test suite for external volumes was created. A script is used to delete all the external volumes after all the tests have run.

To run:
```
docker run -it --rm --ipc=host \
  -v `pwd`:/dcos-ui \
  -e CLUSTER_URL=$OPEN_CLUSTER \
  -e REGION=$REGION \
  -e ACCESS_KEY_ID=$ACCESS_KEY_ID \
  -e SECRET_ACCESS_KEY=$SECRET_ACCESS_KEY \
  mesosphere/dcos-ui dcos-system-test-driver -v --only "external-volumes"  \
  ./system-tests/driver-config/dev.sh
```

`REGION`: AWS Region
`ACCESS_KEY_ID`: AWS Access Key ID
`SECRET_ACCESS_KEY`: AWS Secret Access Key
~~`NUM_VOLS_DELETE`: Number of Volumes to Delete (More or less corresponds to the number of test cases)~~

See `IntegrationTests` secure notes for credentials. `REGION` is as it appears in the `CLUSTER_URL`.

To test:

- Run the above command and verify the results of the test cases
- Observe that the volumes are deleted from the "Removing XX Volumes in Region..." message during the teardown phase
